tests/test_testing.py

```diff
@@ -26,3 +26,10 @@ def test_assert_yaml_fails(caplog):
     obs = log[2].replace("\n", "")  # strip newlines from observed log message
     exp = r"Error parsing file while scanning a simple key.*, line 4, column 1"
     assert re.match(exp, obs)
+
+
+def test_special_character():
```

Comment:
I would call this "hidden character".
```suggestion
def test_hidden_character():
```